package.json

```diff
@@ -23,9 +23,11 @@
         "webpack-dev-server": "3.11.0"
     },
     "dependencies": {
+        "@types/react-svg-pan-zoom": "3.3.1",
```

Comment:
The types should be a dev dependency.

Reply:
I tried to install this as a dev dependency, and I couldn't get it to compile. Maybe you will have more luck.